src/core/server/logging/logging_config.ts

```diff
@@ -140,13 +140,6 @@ export class LoggingConfig {
         layout: { kind: 'pattern', highlight: true },
       } as AppenderConfigType,
     ],
-    [
-      'file',
```

Comment:
Removed since it requires defining a file name. I don't think we should set a default for it.